pages/common/pip3.md

```diff
@@ -0,0 +1,32 @@
+# pip3
+
+> Python package manager.
+> More information: <https://pip.pypa.io>.
+
+- Install a package:
+
+`pip3 install {{package_name}}`
+
+- Install a specific version of a package:
+
+`pip3 install {{package_name}}=={{package_version}}`
+
+- Upgrade a package:
+
+`pip3 install --upgrade {{package_name}}`
+
+- Uninstall a package:
+
+`pip3 uninstall {{package_name}}`
+
+- Save installed packages to file:
```

Comment:
to _a_ file

Reply:
I would rephrase this as:

    - Save the list of installed packages to a file:

Reply:
Thanks I just updated it incorporating your suggestions

Reply:
Hey! I don't see the changes here. Maybe they didn't go through?

Reply:
@volt4ire - You have updated the wrong line. :)

Reply:
Whoops sorry about that I just fixed it